goentri.com.fourthwall.json

```diff
@@ -8,102 +8,102 @@
   "description": "Allows user to easily set up domain using Entri",
   "syncPubKeyDomain": "goentri.com",
   "syncRedirectDomain": "api.goentri.com, goentri.com",
-  "variableDescription": "ip is the website server ip, zendeskverification is the text verification value for zendesk, sendgridHost is sendgrid dynamic host.",
+  "variableDescription": "ip is the website server ip, zendeskverification is the text verification value for zendesk, sendgridHost is sendgrid dynamic host., fourthwallHost is the fourthwall host name to be added, supportHost is the support host being added by fourthwall",
   "sharedProviderName": true,
   "records": [
     {
       "type": "A",
-      "host": "@",
+      "host": "%fourthwallHost%",
       "pointsTo": "%ip%",
       "ttl": 3600
     },
     {
       "type": "CNAME",
-      "host": "www",
-      "pointsTo": "@",
+      "host": "%redirectValue%",
```

Comment:
Not quite sure what would come into this host variable?

Reply:
Is this variable to take "www" or alike? `%redirectValue%` may be a bit misleading as a name. `%redirectSubdomain%` is likely more appropriate.

Reply:
Got it, makes sense. I'm opening another PR, with the correct changes, but focused only on a "support subdomain" template. I'll skip changing this one.